### PR TITLE
68983: Added example to prereqs

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/68983/add-example-command-to-velero-prereqs

Previews:

- https://deploy-preview-943--replicated-docs.netlify.app/enterprise/snapshots-configuring-hostpath#prerequisites

### CODE REVIEW

File: docs/enterprise/snapshots-configuring-nfs.md [hunk 1]
```diff
@@ -12,9 +12,16 @@ Complete the following items before you perform this task:
...
+* The NFS server must be configured to allow access from all of the nodes in the cluster.
+* The NFS directory must be writable by the user:group 1001:1001 from all of the nodes in the cluster.
+  
+  **Example: Changing user:group Permissions on Linux**
+
+   ```bash
+  sudo chown -R 1001:1001 /backups
+  ```
 * Ensure that you configure the user:group 1001:1001 permissions for the directory on the NFS server.
+
```
Comment (sgalsaleh) [inline docs/enterprise/snapshots-configuring-nfs.md:24]: is this extra line intentional?

**Review by sgalsaleh — DISMISSED** [review]

File: docs/enterprise/snapshots-configuring-nfs.md (hunk 1 above)
Comment (jonquil2002) [reply docs/enterprise/snapshots-configuring-nfs.md:24]: I like to add them for readability in the code sometimes, it typically doesn't change the output. But I just removed it in this case.

File: docs/enterprise/snapshots-configuring-nfs.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ Complete the following items before you perform this task:
 
 * Review the limitations and considerations. See [Limitations and Considerations](snapshots-understanding#limitations-and-considerations) in _How to Set Up Backup Storage_.
 * Install the Velero CLI. See [Installing the Velero CLI](snapshots-velero-cli-installing).
-* The NFS server must be configured to allow access from all the nodes in the cluster.
+* The NFS server must be configured to allow access from all of the nodes in the cluster.
 * The NFS directory must be writable by the user:group 1001:1001.
-* Ensure that you configure the user:group 1001:1001 permissions for the directory on the NFS server.
-* All the nodes in the cluster must have the necessary NFS client packages installed to be able to communicate with the NFS server. For example, the `nfs-common` package is a common package used on Ubuntu.
+ * Ensure that you configure the user:group 1001:1001 permissions for the directory on the NFS server.
```
Comment (sgalsaleh) [inline docs/enterprise/snapshots-configuring-nfs.md:17]:
```suggestion
* Ensure that you configure the user:group 1001:1001 permissions for the directory on the NFS server.
```

**Review by sgalsaleh — APPROVED** [review]

**Review by paigecalvert — APPROVED** [review]
LG! Only minor suggestion is that, where the example is placed, it sort of makes it look like the paragraphs that follow are a part of the example, rather than applying at a high level to the requirement itself. Perhaps the example could go after all the paragraphs?

**Pull request merged**